website/cue/reference/components.cue

```diff
@@ -350,7 +350,7 @@ components: {
 					enabled: bool
 
 					if enabled {
-						default: #EncodingCodec | null
+						batched: bool | *false
```

Comment:
`encoding.codec` controls the event, and sometimes the batch, encoding so I added this flag to template the descriptions for the encoding differently.